### PR TITLE
E2E: adding selenium task

### PR DESCRIPTION
**Pull request by joesepi**

### CODE REVIEW

**Comment by mrjoelkemp** [comment]
We need to add those new selenium-based dependencies to the package.json no? Try doing an `npm install --save aNewSeleniumBasedDep` to have it modify the package.json file automatically

**Comment by kaicataldo** [comment]
Wondering the same thing ^^

**Comment by mrjoelkemp** [comment]
Ah, I see, they're on https://github.com/behance/be.net/pull/11105/files for npm3. Still seems like they should be declared here and will get deduped during an install within the consumer

**Comment by joesepi** [comment]
good call.

File: tasks/selenium.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = function(grunt) {
+  grunt.loadNpmTasks('grunt-webdriver');
```
Comment (mrjoelkemp) [inline tasks/selenium.js]: For uniformity, we should use https://github.com/behance/grunt-behance/blob/master/tasks/jscs.js#L1

**Comment by mrjoelkemp** [comment]
Shouldn't we also add this new task to one of the aliases so it runs? https://github.com/behance/grunt-behance/blob/master/tasks/aliases.js

**Comment by mikesherov** [comment]
I think we should not put this in grunt-behance and actively move away from grunt-behance.

**Comment by mikesherov** [comment]
My suggestion was to put this task in the taks dir on the network codebase.

**Comment by joesepi** [comment]
The tasks dir seems to be abandoned. I would be happy to not put it in grunt-behance becuase I am getting a number of Peer dependency issues in that repo. :(

**Comment by mrjoelkemp** [comment]
Even if we move the task definition out, we need to be able to invoke it either through one of the aliases or from a consumer calling `grunt selenium-the-things`

**Comment by mikesherov** [comment]
This is just moving this code to behamce tasks dir. Please do that.

**Pull request closed without merging**